src/libs/runtime-utils.js

```diff
@@ -77,14 +87,33 @@ export const getValidGpuTypes = (numCpus, mem, zone) => {
   return validGpuTypes || { name: '?', type: '?', numGpus: '?', maxNumCpus: '?', maxMem: '?', price: NaN, preemptiblePrice: NaN }
 }
 
-const gpuCost = (gpuType, numGpus) => _.find({ type: gpuType, numGpus }, gpuTypes)?.price || NaN
-
 const dataprocCost = (machineType, numInstances) => {
   const { cpu: cpuPrice } = findMachineType(machineType)
 
   return cpuPrice * numInstances * dataprocCpuPrice
 }
 
+const getHourlyCostForMachineType = (machineTypeName, region, isPreemptible) => {
+  const { cpu, memory } = _.find({ name: machineTypeName }, machineTypes)
+  const { n1HourlyCpuPrice, preemptibleN1HourlyCpuPrice, n1HourlyGBRamPrice, preemptibleN1HourlyGBRamPrice } = _.find({ name: _.toUpper(region) },
+    regionToPrices)
+  return !isPreemptible ?
+    (cpu * n1HourlyCpuPrice) + (memory * n1HourlyGBRamPrice) :
+    (cpu * preemptibleN1HourlyCpuPrice) + (memory * preemptibleN1HourlyGBRamPrice)
+}
+
+const getGpuCost = (gpuType, numGpus, region) => {
+  const prices = _.find({ name: region }, regionToPrices)
+  const price = Utils.switchCase(gpuType,
```

Comment:
Would this be assuming too much about naming?
```js
const price = prices[`${_.last(_.split('-', gpuType))}HourlyPrice`]
```

Reply:
Done. I think when gpus that aren't nvidia are supported by Google Cloud we'll have to do something else. But for now, this makes sense.